src/version/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+
+find_package(Git QUIET)
+
+if(Git_FOUND AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../../.git")
+    execute_process(COMMAND ${GIT_EXECUTABLE} fetch --all --tags --force)
+    execute_process(COMMAND ${GIT_EXECUTABLE} describe --exclude latest --abbrev=0
```

Comment:
again not sure it's good idea to use annotated tags over lightweight + what happens for builds several commits after tagged version

Reply:
See latest release in my fork. Increasing number is added after dash.